### PR TITLE
feat: add option to set startingDeadlineSeconds for InfluxDB backup

### PR DESCRIPTION
**Pull request by jakubkrzywda**
Currently, there is no option to set the `startingDeadlineSeconds` for the InfluxDB backup job, which would be useful in situations when the backup CronJob Controller is disabled for a longer period and then enabled again. With the current setting, according to the Kubernetes [documentation](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/#cron-job-limitations), if there are more than 100 missed schedules, then it does not start the job anymore. This PR exposes the configuration parameter in `values.yaml` file to enable more accurate configuration of the backup procedure.

- [ ] CHANGELOG.md updated
- [ ] Rebased/mergable
- [ ] Tests pass
- [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: charts/influxdb/values.yaml
Comment (rawkode) [inline charts/influxdb/values.yaml:270]: Needs a default

Comment (jakubkrzywda) [reply charts/influxdb/values.yaml]: Fixed!

**Comment by rawkode** [comment]
Sorry @jakubkrzywda shouldn't the default be a number? 👍

**Comment by jakubkrzywda** [comment]
> Sorry @jakubkrzywda shouldn't the default be a number?

@rawkode based on [the Kubernetes documentation](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/#cron-job-limitations) and other implementation (for example [elasticsearch-curator](https://github.com/helm/charts/blob/master/stable/elasticsearch-curator/values.yaml#L14)), I would say it should by default be left unset.

In our InfluxDB deployment we set it to 200 seconds, but that is dependent on the frequency of related schedules. However, I am afraid that setting such a value as a default one may confuse other users, who may expect the default behavior to be inline with Kubernetes.

**Comment by rawkode** [comment]
OK. Perfect. You're correct, I was mistaken 🙂

**Review by rawkode — DISMISSED** [review]

**Review by timhallinflux — DISMISSED** [review]

**Comment by jakubkrzywda** [comment]
I have rebased the branch and bumped the version.

**Review by timhallinflux — DISMISSED** [review]

**Comment by jakubkrzywda** [comment]
@rawkode is there any way we could make the merging process more smooth? I got this PR approved but then I had to bump the Chart version and wait for approvals once again. Now I see that the version changed again and I'm still waiting for the second approval... Or maybe I'm doing something wrong? Please let me know how we can get this and https://github.com/influxdata/helm-charts/pull/144 merged.

**Comment by rawkode** [comment]
Yeah, it's not ideal. We're using the official Helm test and release GitHub Actions; but it's pretty painful.

We could disable auto-release and instead release the charts once a week; this would make it a lot easier for people contributing

**Comment by rawkode** [comment]
I'll get these PRs reviewed and merged tomorrow. Thanks for your patience

**Comment by jakubkrzywda** [comment]
> I'll get these PRs reviewed and merged tomorrow. Thanks for your patience

Thanks! Let me know if I can help anyhow with the process.

**Comment by jakubkrzywda** [comment]
> > I'll get these PRs reviewed and merged tomorrow. Thanks for your patience
> 
> Thanks! Let me know if I can help anyhow with the process.

@rawkode any updates on this? can I help somehow in getting these two PRs merged?

**Comment by rawkode** [comment]
@jakubkrzywda I sent you PRs to each of your open PRs to fix the version mismatch

**Comment by jakubkrzywda** [comment]
> @jakubkrzywda I sent you PRs to each of your open PRs to fix the version mismatch

Thanks @rawkode! Merged for this PR and https://github.com/influxdata/helm-charts/pull/144

**Review by rawkode — APPROVED** [review]

**Comment by rawkode** [comment]
@jakubkrzywda thanks for your patience. I merged a PR yesterday that removes the requirement for contributors to bump the version, this means we won't need to consistently ask for contributors to do this, and PRs won't need version bumps due to conflicts.

The maintainers will bump versions for releases around once a week, depending on contributions

**Review by timhallinflux — APPROVED** [review]

**Review by rawkode — APPROVED** [review]

**Comment by rawkode** [comment]
Merged manually 👍🏻

**Comment by jakubkrzywda** [comment]
> Merged manually 👍🏻

Thanks @rawkode for your help on the way!

**Pull request closed without merging**
